tjmonopix2/testbench.yaml

```diff
@@ -1,5 +1,5 @@
 general: # General configuration
-  readout_system: # Readout system, available platforms are BDAQ53 or MIO3 (+ GPAC). BDAQ53 is default
```

Comment:
I think we can delete the whole key (it should be identical, software-wise in a future BDAQcore version)